Instructions/movement_instruction.py

```diff
@@ -0,0 +1,32 @@
+from collections import deque
+
+# Ours
+from ..Tasks.movement_task import MovementTask
+from instruction_base import InstructionBase
+from ..Utilities import constants as c
+
+class MovementInstruction(InstructionBase):
+
+    def __init__(self, data):
+        super(MovementInstruction, self).__init__()
+        # The data here will eventually be the raw binary data that
+        # has traveled over the network. The decoding may be more
+        # involved than what is shown here.
+        (self.x, self.y, self.z) = data
+        
+        self.movementTask = None
+
+    def get_task(self, drone):
+        queue = deque()
+        if self.x != 0:
```

Comment:
Not sure how I feel about this function, it would be fairly easy to just send movement as a vector here rather than encoding each directional axis separately

Reply:
I don't disagree. It would require changing how MovementTask works, because it currently relies on a queue of movements along the directional axes. It wouldn't be that hard to change but I think that it should be the topic of a future pull request because I anticipate that we are going to be doing a major overhaul on the lower-level implementation of flight soon. The Task and Instruction classes will be the framework we put the lower-level implementation in.